packages/python/patch-no-access

```diff
@@ -0,0 +1,14 @@
+--- Modules/posixmodule.c
++++ Modules/posixmodule.c
+@@ -2620,7 +2620,11 @@
+     }
+     else
+ #endif
++#ifdef HAVE_ACCESS
+         result = access(path->narrow, mode);
++#else
++        {result = -1; errno = EFAULT;}
```

Comment:
I chose the error code semi-randomly, to be 'unusual` so it would be easier to cross correlate exceptions with lines of code. I'll change them to be ENOSYS.